### ISSUE TITLE
Feature: Add Plex update.sh once symlinks are created. 

### ISSUE THREAD

**Issue by skearney414** [sureshfizzy/CineSync#38]
Similar to zurgs on_library_update, might be a good idea to have this on symlink creation ? 

I have added it in myself but would be great to have it built in

**Comment by sureshfizzy** [comment]
> Similar to zurgs on_library_update, might be a good idea to have this on symlink creation ?
> 
> I have added it in myself but would be great to have it built in

Is this for updating the Plex library after fetching the new file ? What's the current behaviour without update.sh.

**Comment by skearney414** [comment]
Yeah so once zurg detects a new file it triggers on_library_updated which can be set on the config.yml. This typically calls plex_update.sh which you can find in the zurg repo, it accepts a file path as a parameter and triggers a sync just for that file which means the plex library gets the file straight away.

Currently you would have to wait for a scan to run in plex or manually run a scan without this feature.

To add it, it was as simple as kicking of a bash subprocess after symlink that calls plex_update with the destination dir

Basically this whole process means that it works with plex debrid. Plex debrid adds the file, cinesync picks up the file and refreshes the plex library

**Comment by sureshfizzy** [comment]
> Yeah so once zurg detects a new file it triggers on_library_updated which can be set on the config.yml. This typically calls plex_update.sh which you can find in the zurg repo, it accepts a file path as a parameter and triggers a sync just for that file which means the plex library gets the file straight away.
> 
> Currently you would have to wait for a scan to run in plex or manually run a scan without this feature.
> 
> To add it, it was as simple as kicking of a bash subprocess after symlink that calls plex_update with the destination dir
> 
> Basically this whole process means that it works with plex debrid. Plex debrid adds the file, cinesync picks up the file and refreshes the plex library

i'll check upon this and drop some features accordingly

**Comment by sureshfizzy** [comment]
@skearney414, Update env and pull new image it will be good.

**Comment by sureshfizzy** [comment]
> Fantastic, thank you :) just wondering will this also kick off on initial run? Just that if your initial run is for a lot of files it might crash plex? Thanks again

give me a min, i did a mistake.

also, as per the new logic, it will update plex only when symlink is created

**Comment by sureshfizzy** [comment]
@skearney414 good to pull now
